tests/test_intl.py

```diff
@@ -15,64 +15,64 @@
 import re
```

Comment:
Review is **NOT** needed.
- Converting yield testing into pytest.mark.parametrize
- Replacing calling 'msgfmt' command with babel code.